### PR TITLE
DT-1290: Add libva, intel-gmm and intel-media-driver

### PR DESCRIPTION
**Pull request by sergio-costas**
This patch adds the libAV library with DRI3 support, the Intel Graphics Memory Manager library (needed for the driver) and the Intel Media Driver.

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -1427,8 +1427,62 @@ parts:
         sed -i 's#includedir=/usr#includedir=${prefix}#' $PC
       done
 
+  libva:
+    after: [ libtool, meson-deps, libayatana-appindicator ]
```
Comment (kenvandine) [inline snapcraft.yaml:1431]: Does libva really need to be after libayatana-appindicator?

**Review by kenvandine — CHANGES_REQUESTED** [review]

File: snapcraft.yaml (hunk 1 above)
Comment (sergio-costas) [reply snapcraft.yaml:1431]: Not really... it was just to build it "in order".

**Comment by sergio-costas** [comment]
Anyway, as I see, firefox still uses gnome-3-38...

**Review by kenvandine — APPROVED** [review]
Thanks, this is good to land

**Pull request merged**